airflow_provider_fivetran/hooks/fivetran.py

```diff
@@ -103,137 +105,188 @@ def _do_api_call(self, endpoint_info, json=None, force=''):
-        URL_SETUP = "https://fivetran.com/dashboard/connectors/{}/{}/setup".format(
-            connector_details["service"], connector_details["schema"]
-        )
+        service_name = connector_details["service"]
+        schema_name = connector_details["schema"]
         setup_state = connector_details["status"]["setup_state"]
+
         if setup_state != "connected":
-            EXC_SETUP: str = (
-                'Fivetran connector "{}" not correctly configured, status: {}; '
-                + "please complete setup at {}"
+            raise AirflowException(
+                f'Fivetran connector "{self.connector_id}" not correctly configured, '
+                f"status: {setup_state}\nPlease see: "
+                f"{self._connector_ui_url_setup(service_name, schema_name)}"
             )
-            raise ValueError(
-                EXC_SETUP.format(self.connector_id, setup_state, URL_SETUP)
-            )
-        # We need to know the previous job's completion time to know if the job succeeded or failed
-        succeeded_at = self.parse_timestamp(connector_details["succeeded_at"])
-        failed_at = self.parse_timestamp(connector_details["failed_at"])
-        previous_completed_at = succeeded_at if succeeded_at > failed_at else failed_at
         # URL for connector logs within the UI
-        log.info(
-            "Connector type: {}, connector schema: {}".format(
-                connector_details["service"], connector_details["schema"]
-            )
+        self.log.info(
+            f"Connector type: {service_name}, connector schema: {schema_name}"
+        )
+        self.log.info(
+            f"Connectors logs at "
+            f"{self._connector_ui_url_logs(service_name, schema_name)}"
         )
-        log.info("Connectors logs at {}".format(URL_LOGS))
         return resp
 
     def set_manual_schedule(self, connector_id):
-        # Set connector to manual sync mode, required to force sync through the API
-        return self._do_api_call(('PATCH',connector_id),json.dumps({"schedule_type": "manual"}))
+        """
+        Set connector to manual sync mode, required to force sync through the API.
+            Syncs will no longer be performed automatically and must be started
+            via the API.
+        :param connector_id: Fivetran connector_id, found in connector settings
+            page in the Fivetran user interface.
+        :type connector_id: str
+        """
+        return self._do_api_call(
+            ("PATCH", connector_id), json.dumps({"schedule_type": "manual"})
+        )
 
     def start_fivetran_sync(self, connector_id):
-        return self._do_api_call(('POST',connector_id),json=None, force='/force')
+        """
+        :param connector_id: Fivetran connector_id, found in connector settings
+            page in the Fivetran user interface.
+        :type connector_id: str
+        """
+        return self._do_api_call(("POST", connector_id), json=None, force="/force")
```

Comment:
force="/force" is a bit gnarly. Since _do_api_call is meant to receive an endpoint path, we should be building that path here even if it breaks the illusion that we are just passing connector_id.